### ISSUE TITLE
docker tag without leading v

### ISSUE THREAD

**Issue by clux** [swagger-api/swagger-ui#4497]
Out of all the projects on the [front page on docker hub](https://hub.docker.com/explore/), no one is prefixing tags with a leading `v`. Consistency wise, is it possible to tag without it?

**Comment by heldersepu** [comment]
How is this related to swagger-ui?

**Comment by clux** [comment]
https://hub.docker.com/r/swaggerapi/swagger-ui/tags/

**Comment by heldersepu** [comment]
Is that preventing you from accomplish something?

PHP has letters after the number:
https://hub.docker.com/r/library/php/tags/

Node has some non consistent tags too, some start with letter some with numbers:
https://hub.docker.com/r/library/node/tags/

Ubuntu seems to have it's own pattern (name-date):
https://hub.docker.com/r/library/ubuntu/tags/

**Comment by clux** [comment]
Well, parsing tags that are masquerading as semver, as semver.

The ones you listed (apart from ubuntu) all have valid semver tags (dash separated stuff after version is allowed). It's not a big problem, just trying to lock down version pinning so that we don't rely on floating tags everywhere. Semver enforcement is a nice way of doing that.

**Comment by webron** [comment]
I don't have a problem considering it. @shockey, @ponelat  - thoughts?

**Comment by ponelat** [comment]
Yeah no problem my side. Might be worth re-tagging all to the new format ( and keeping the old ones around ), its reasonably easy to do. As I wouldn't want people looking for a version, and having to guess whether it starts with a `v` or not.

**Comment by shockey** [comment]
Agreed @ponelat.

This isn't hard to do technically, we just need to strip the leading v off of the tag given to docker.

**Comment by webron** [comment]
I imagine for new images, it's just a matter of updating our build process. For older ones, we can probably write a script that will run through the images (maybe even keep both to not break any existing dependencies?).

**Comment by shockey** [comment]
Fixed - Swagger-UI and Swagger-Editor will publish without a leading `v` going forward.

Thanks @clux!

**Comment by shockey** [comment]
This has regressed in both projects - probably due to a change in our deployment scripts.

Reopening.
